lib/package-managers/base-package-manager.js

```diff
@@ -11,11 +11,19 @@ exports.BasePackageManager = class {
   }
 
   run(command, args = [], workingDirectory = process.cwd()) {
+    const isWindows = process.platform === "win32";
```

Comment:
Please move this line to the beginning of the file, so it only needs to calculate once.